module/modmanager/manager.go

```diff
@@ -991,6 +995,7 @@ func (m *module) dial() error {
 	// out.
 	m.sharedConn.ResetConn(rpc.GrpcOverHTTPClientConn{ClientConn: conn}, m.logger)
 	m.client = pb.NewModuleServiceClient(m.sharedConn.GrpcConn())
+	m.robotClient = robotpb.NewRobotServiceClient(m.sharedConn.GrpcConn())
```

Comment:
Would like a more-context-than-I-have opinion on whether we should use a single `m.sharedConn.GrpcConn()` vs. two different calls as I have here